torch/csrc/Module.cpp

```diff
@@ -709,6 +717,11 @@ PyObject* initModule() {
   torch::python::init_bindings(module);
 #ifdef USE_CUDA
   torch::cuda::initModule(module);
+  torch::cuda::shared::initCudartBindings(module);
```

Comment:
Could these be part of the cuda initModule? Or you think it's clearer to add them here explicitly?

Reply:
I guess I could do that too, no strong preference.